src/plugins/flow.js

```diff
@@ -726,6 +730,21 @@ pp.typeCastToParameter = function (node) {
   );
 };
 
+pp.flowParseVariance = function() {
+  let variance = null;
+  if (this.match(tt.plusMin)) {
+    variance = this.startNode();
+    if (this.state.value === "+") {
+      variance.kind = "plus";
+    } else if (this.state.value === "-") {
+      variance.kind = "minus";
+    }
+    this.eat(tt.plusMin);
```

Comment:
Will change the `eat(plusMin)` to `next()`, which will hopefully clarify the intent better. The tests do exercise the node positioning, so regression is unlikely here.